authors/apps/articles/tests/test_bookmark_article.py

```diff
@@ -0,0 +1,71 @@
+import os
+from django.urls import reverse
+from rest_framework import status
+from .article_tests_base_class import ArticlesBaseTest
+from .test_data import *
+from authors.apps.articles.views import *
+
+
+class BookmarkArticleTestCase(ArticlesBaseTest):
+    """Test favoriting an article functionality"""
```

Comment:
@ElMonstro, does this PR do favoriting functionality?

Reply:
I believe this docstring is misleading, could you please correct it